packages/css/src/components/description-list/description-list.scss

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@import "../../common/breakpoint";
+@import "../../common/text-rendering";
+
+@mixin reset {
+  box-sizing: border-box;
+  margin-block: 0;
+  margin-inline: 0;
+}
+
+.ams-description-list {
+  color: var(--ams-description-list-color);
+  display: grid;
+  font-family: var(--ams-description-list-font-family);
+  font-size: var(--ams-description-list-font-size);
+  font-weight: var(--ams-description-list-font-weight);
+  gap: var(--ams-description-list-gap);
+  grid-template-columns: 1fr;
+  line-height: var(--ams-description-list-line-height);
+
+  @media screen and (min-width: $ams-breakpoint-medium) {
+    grid-template-columns: 1fr 2fr;
+  }
+
+  @include reset;
+}
+
+.ams-description-list--inverse-color {
+  color: var(--ams-description-list-inverse-color);
+}
+
+.ams-description-list__term {
+  @include reset;
+  @include text-rendering;
+}
+
+.ams-description-list__details {
+  font-weight: var(--ams-description-list-details-font-weight);
+  padding-inline-start: var(--ams-description-list-details-padding-inline-start);
+
+  @media screen and (min-width: $ams-breakpoint-medium) {
+    padding-inline-start: 0;
+  }
+
+  @media screen and (min-width: $ams-breakpoint-medium) {
+    grid-column-start: 2;
+  }
+
+  @include reset;
```

Comment:
`details` already gets `box-sizing: border-box` from the parent I think, it only needs `margin-inline: 0;`

Reply:
`margin-inline: 0;` is missing after your change

Reply:
It's on line 12

Reply:
It shouldn't be part of the general reset though, it should be part of a `reset-details` mixin. The default inline margin currently isn't reset